apps/playground/src/app/components/popovers/popovers/popovers.component.html

```diff
@@ -5,20 +5,41 @@
       type="button"
       [skyPopover]="myPopover0"
     >
-      Popover demo on click
+      Open popover with interactive content
     </button>
 
     <button
```

Comment:
Let's leave this here until we actually remove the capability. Having this in place in the playground might be useful if an issue ever comes up